deployment/pypi/install.ps1

```diff
@@ -0,0 +1,48 @@
+param([bool]$version_ts=$false)
+$CWD = $PWD
+
+$OS_SPEC = "windows"
+$WHEEL_SPEC = "win_amd64"
+
+$TIME_STAMP = date -u "+%y%m%d%H%M"
+$NNI_VERSION_VALUE = git describe --tags --abbrev=0
+
+# To include time stamp in version value, run:
+# make version_ts=true build
+
+if($version_ts){
+    $NNI_VERSION_VALUE = "$NNI_VERSION_VALUE.$TIME_STAMP"
+}
+
+$NNI_VERSION_TEMPLATE = "999.0.0-developing"
+
+python -m pip install --user --upgrade setuptools wheel
+
+$nodeUrl = "https://aka.ms/nni/nodejs-download/win64"
+$NNI_NODE_TARBALL = "$CWD\node-$OS_SPEC-x64.tar.gz"
+$NNI_NODE_FOLDER = "$CWD\node-$OS_SPEC-x64"
+(New-Object Net.WebClient).DownloadFile($nodeUrl, $NNI_NODE_TARBALL)
+if(Test-Path $NNI_NODE_FOLDER){
+    Remove-Item $NNI_NODE_FOLDER -Recurse -Force
+}
+New-Item $NNI_NODE_FOLDER -ItemType Directory
+cmd /c tar -xf $NNI_NODE_TARBALL -C $NNI_NODE_FOLDER --strip-components 1
+
+cd $CWD\..\..\src\nni_manager
+yarn
+yarn build
+cd $CWD\..\..\src\webui 
+yarn
+yarn build
+if(Test-Path $CWD\nni){
+    Remove-Item $CWD\nni -r -fo
+}
+Copy-Item $CWD\..\..\src\nni_manager\dist $CWD\nni -Recurse
+Copy-Item $CWD\..\..\src\webui\build $CWD\nni\static -Recurse
```

Comment:
Error "Copy-Item : Cannot find path 'C:\XXXX\XXXX\nni\src\webui\build' because it does not exist."
Shall we create the directory first?

Reply:
Is there a new generated directory in nni\src\webui\build? If it webui build successfully, it should be there and will be found by Copy-Item.

Reply:
tar problem is fixed, please check

Reply:
Thank you for your help. I find this problem is fixed now.